ansible/roles/compiled/templates/astro/calceph.lua

```diff
@@ -0,0 +1,16 @@
+-- -*- lua -*-
+help([[
+This module configures calceph {{ version_number }} for use
+]])
+whatis("Version: {{ version_number }}")
+whatis("Keywords: calceph")
+whatis("Description: calceph {{ version_number }}")
+
+setenv("PSRHOME_CALCEPH_PATH", "{{ install_dir }}")
+
+prepend_path('PATH', "{{ install_dir }}/bin")
+prepend_path('LD_LIBRARY_PATH', "{{ install_dir }}/lib")
+prepend_path('LIBRARY_PATH', "{{ install_dir }}/lib")
```

Comment:
I googled these to try and understand them. I understand it probably doesn't make a difference, but do we really need to do both LD_LIBRARY_PATH and LIBRARY_PATH? From what I can see it looks like the one is only for for compilation. 

https://stackoverflow.com/questions/4250624/ld-library-path-vs-library-path

Reply:
on the other hand, I suppose adding everything that could be needed is probably safer.

Reply:
Yeah, I do it mostly for the safety aspect. Sometimes people compile code! And all the variations on INCLUDE / CPATH are annoying too and mostly because I've encountered issues at some stage when compiling something else :-/.